src/apis/chat/queries.ts

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query';
+import { ChatEndPoint } from '.';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import {
+  GetChatMessageErrorHandler,
+  GetChatRoomErrorHandler,
+  PostChatRoomErrorHandler,
+} from './error';
+
+// 채팅방 생성 쿼리
+export function ChatRoomQuery() {
+  const { data, isError, error } = useQuery({
+    queryKey: ['make-chat-room'],
+    queryFn: () => ChatEndPoint.postChatRoom(),
+  });
+  if (isError && axios.isAxiosError(error)) {
+    toast.error(PostChatRoomErrorHandler(error));
+    return;
+  }
+  return data;
+}
```

Comment:
`CreateChatRoomMutate`로 하고, `Query`말고, `useMutation`을 사용해서 구현하는게 좋아보입니다.

```suggestion
export function CreateChatRoomMutate() {
  const queryClient = useQueryClient();
  return useMutation({
    mutationFn: ChatEndPoint.postMakeChatRoom,
    onSuccess: () => {
      queryClient.invalidateQueries({
        queryKey: ['make-chat-room'],
      });
    },
    onError: (error: AxiosError) => {
      toast.error(PostChatRoomErrorHandler(error));
    },
  });
}
```